gr-blocks/python/blocks/qa_file_source_sink.py

```diff
@@ -104,6 +105,67 @@ def test_file_source_can_seek_after_open(self):
 
         os.remove(filename)
 
+    def test_begin_tag(self):
+        src_data = range(1000)
+        expected_result = range(1000)
+
+        filename = "tmp.32f"
```

Comment:
We have lots of bad habits strewn throughout the tree, which unfortunately get copied by newcomers.  So, go ahead and do it the correct way for this new test, and for extra credit, submit a new PR with changes to the other ones.